### PR TITLE
Enable inverse squared falloff on Enlighten and Deprecated Realtime Enlighten

### PR DESCRIPTION
**Pull request by sebastienlagarde**
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=master&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

Alternative, launch Yamato (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?

**Halo Effect**: None, Low, Medium, High?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs [hunk 1]
```diff
@@ -581,6 +585,14 @@ bool SetRenderingFeatures()
             {
                 Debug.LogError("High Definition Render Pipeline doesn't support Gamma mode, change to Linear mode");
             }
+
+            if ( UnityEditor.Lightmapping.realtimeGI)
+            {
+#if UNITY_2019_3_OR_NEWER
+#elif UNITY_2019_2_OR_NEWER
+                Debug.LogWarning("Enlighten for HDRP will be deprecated in 2019.3, please prefer to use Progressive CPU or Progressive GPU mode");
```
Comment (chikuba) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs]: Not sure this sounds correct. "please use Progressive CPU or Progressive GPU instead"

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs]: ok, will update. Thanks

**Comment by sebastienlagarde** [comment]
Katana is green, merging.

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/GlobalIlluminationUtils.cs [hunk 2]
```diff
@@ -175,23 +175,28 @@ public static bool LightDataGIExtract(Light l, ref LightDataGI ld)
...
 #else
-                    ld.InitNoBake(l.GetInstanceID());
+                LightMode mode = LightmapperUtils.Extract(l.bakingOutput.lightmapBakeType);
 #endif
 
-                    lightsOutput[i] = ld;
-                }
-            };
+                if (mode == LightMode.Realtime)
+                    ld.InitNoBake(l.GetInstanceID());
```
[the remark this answers is not in the document]
Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/GlobalIlluminationUtils.cs]: Sorry I don't understand, 

    public enum LightMode : byte
    {
        Realtime = 0,
        Mixed = 1,
        Baked = 2,
        Unknown = 3
    }

=> Realtime here, mean no realtime GI, so this is expected right?

**Pull request merged**